test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,60 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+     let svg = TestMethods.generateSVG(400, 100);
+      scale = new Plottable.Scales.Time();
```

Comment:
The scale is already initialized in the `beforeEach()` block, on line 7. No need to do it here as well